code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/model/SubmissionsGraph.kt

```diff
@@ -55,7 +63,11 @@ class GraphBuilder<T>(private val submissionsGraphContext: SubmissionsGraphConte
     fun build(): SubmissionsGraph = SubmissionsGraph(graph)
 }
 
-fun <T> buildGraph(context: SubmissionsGraphContext<T>, block: GraphBuilder<T>.() -> Unit): SubmissionsGraph {
-    val builder = GraphBuilder(context)
+fun <T> buildGraph(
+    context: SubmissionsGraphContext<T>,
+    graph: SubmissionsGraphAlias = SimpleDirectedWeightedGraph(SubmissionsGraphEdge::class.java),
```

Comment:
I am a little confused since the function's name is 'buildGraph', but we pass the graph...